sdk/state/close.go

```diff
@@ -57,32 +60,37 @@ func amountToResponder(balance int64) int64 {
 	return 0
 }
 
-// ProposeCoordinatedClose proposes a close transaction to be submitted immediately.
-// This should be used when participants are in agreement on the final txClose parameters, but would
-// like to submit earlier than the original observation time.
-func (c *Channel) ProposeCoordinatedClose() (CloseAgreement, error) {
+// ProposeClose proposes that the latest authorized close agreement be submitted
+// without waiting the observation period. This should be used when participants
+// are in agreement on the final close state, but would like to submit earlier
+// than the original observation time.
+func (c *Channel) ProposeClose() (CloseAgreement, error) {
 	d := c.latestAuthorizedCloseAgreement.Details
 	d.ObservationPeriodTime = 0
 	d.ObservationPeriodLedgerGap = 0
 
-	_, txCoordinatedClose, err := c.CloseTxs(d)
+	_, txClose, err := c.CloseTxs(d)
 	if err != nil {
 		return CloseAgreement{}, fmt.Errorf("making coordianted close transactions: %w", err)
```

Comment:
the only reason I found this is bc I misspelled coordinated when doing global search again 😅  
```suggestion
		return CloseAgreement{}, fmt.Errorf("making close transactions: %w", err)
```